server/api/controllers/invoice-item.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var InvoiceItem = require('../models/invoice-item');
+var Invoice = require('../models/invoice');
+var _ = require('lodash');
+var R = require('ramda');
+
+var invoiceItemAPI = {
+  invoiceItem: {},
+
+  createInvoiceItemObj: function(req) {
+    var itemObject = new InvoiceItem({
+      invoiceId: req.body.invoiceId,
+      description: req.body.description,
+      qty: req.body.qty,
+      rate: req.body.rate,
+      isFlatFee: req.body.isFlatFee
+    });
+
+    return itemObject;
+  },
+
+  createInvoiceItem: function(invoiceId, itemObj, options, cb) {
```

Comment:
let's change all the `item` references to `invoiceItem` so it will be more descriptive.